### ISSUE TITLE
Remove email address not possible

### ISSUE THREAD

**Issue by dododedodonl** [kreait/firebase-php#459]
**Describe the bug**
Once an email address is set, it is not possible to remove it.

**To Reproduce using the attributes**
```php
$firebaseAuth->updateUser($uid, ['email'=>null]);
// or
$firebaseAuth->updateUser($uid, ['unverifiedemail'=>null]);
// or
$firebaseAuth->updateUser($uid, ['verifiedemail'=>null]);
```

Results in `TypeError: Argument 1 passed to Kreait/Firebase/Value/Email::__construct() must be of the type string, null given, called in vendor/kreait/firebase-php/src/Firebase/Request/EditUserTrait.php on line 132`

**To Reproduce using the delete attribute**
```php
$firebaseAuth->updateUser($uid, ['deleteattribute'=>'email']);
```

Results in a successful query to the firebase api, without updating / removing anything, the email address is still on the returned `UserRecord`.

**Expected behavior**
`email` attribute to null
`email_verified` attribute to false

**Environment (please complete the following information):**
 - OS: MacOS 10.14.6
 - PHP version: 7.4.8
 - Firebase SDK Version: kreait/firebase-php 5.7.0

**Additional context**
The firebase api will remove the email address if it is supplied in the `deleteAttribute` array on the `UpdateUser` request.

**Comment by jeromegamez** [comment]
Thank you for the detailed issue description 🙌.

This is a property/limitation of Firebase itself and can't be solved in code on the SDK side: as far as I know, it is only possible to change an email address, not to remove it. The only removable attributes are the photo url and the display name.

If you can provide me with documentation or an implementation of working email removal (preferably in the official SDKs, where I couldn't find one), please reopen the issue and/or add another comment pointing me towards it, so I can look into it, thanks!

**Comment by jeromegamez** [comment]
You're fast 😅

**Comment by jeromegamez** [comment]
Implemented with 3af9961fc82774c112138645026a79f8af8e633f
